src/index.js

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import reducer from './redux/reducers';
 
 import './assets/stylesheets/reset.css';
 import './assets/stylesheets/index.css';
 import App from './reactjs/components/App';
 import reportWebVitals from './reportWebVitals';
 
+const store = configureStore({
+  reducer,
+});
+
```

Comment:
Please ensure that you move your `store` into the redux folder according to the project requirement. You should have something like the example below:  
````
/src/redux/configureStore.js
````

Then you should Import `Reducers` to the : 
````
/src/redux/configureStore.js
````

Then use `combineReducers` to create a reducer.

* `booksReducer` and `categoriesReducer` should be added to `combineReducers`. according to the project requirement.

Reply:
done